models/stock_picking.py

```diff
@@ -61,20 +61,28 @@ def is_compatible_package(self, package_name):
 
         return res
 
-    def update_picking(self, expected_package_name=None, **kwargs):
+    def update_picking(self, expected_package_name=None, u_transport_id=None, **kwargs):
```

Comment:
Also, why is expected_package_name not in kwargs?

Reply:
This is being handled in branch udes-story/528